docs/persistence.md

```diff
@@ -0,0 +1,110 @@
+# Persistence
+
+Generally speaking, adding database storage or persistence to your Probot App will greatly complicate your life. It's perfectly doable, but for most use-cases you'll be able to manage relevant data within GitHub issues, pull requests and your app's configuration file.
+
+For when you absolutely do need external data storage, here are some examples using a few popular database solutions. Your needs will be slightly different from what any example can help with. Below are some code suggestions to help you get started.
+
+## MongoDB (with Mongoose)
+
+[MongoDB](https://mongodb.com) is a popular NoSQL database, and [Mongoose](http://mongoosejs.com) is the de facto Node.js wrapper for it.
```

Comment:
maybe `a popular` instead of `the de facto`?

Reply:
Yeah that's probably less biased, good call 👍